proto/axelar/axelarnet/v1beta1/tx.proto

```diff
@@ -77,11 +77,17 @@ message AddCosmosBasedChainRequest {
   option (permission.exported.v1beta1.permission_role) = ROLE_CHAIN_MANAGEMENT;
   bytes sender = 1 [ (gogoproto.casttype) =
                          "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
-  nexus.exported.v1beta1.Chain chain = 2 [ (gogoproto.nullable) = false ];
+  nexus.exported.v1beta1.Chain chain = 2 [
+    deprecated = true,
+    (gogoproto.nullable) = false
+  ]; // chain was deprecated in v0.27
   string addr_prefix = 3;
   reserved 4; // min_amount was removed in v0.15
   repeated nexus.exported.v1beta1.Asset native_assets = 5
       [ (gogoproto.nullable) = false ];
+  string cosmos_chain = 6
```

Comment:
```suggestion
  string chain = 6
```
The name of the message `AddCosmosBasedChainRequest` already suggests that the chain is a cosmos-based one. Simpler is better. You can deprecate and rename the other existing one.

Reply:
Want to avoid breaking compatibility of field names as they're used in the JSON representation. Let's rename it to the simpler version when we do the v1beta1 -> v1 change?

Reply:
then leave a TODO so that we don't forget